src/components/TheSidebar.vue

```diff
@@ -7,7 +7,12 @@
 					:key="JSON.stringify(route.to) || route.href"
 					:class="{ 'list-item': true, active: route.active }"
 				>
-					<base-link class="list-content" :to="route.to" :href="route.href">
+					<base-link
+						:href="route.href"
+						class="list-content"
+						:to="route.to || undefined"
```

Comment:
```suggestion
						:to="route.to"
```
isn't `undefined` already the default value?

Reply:
I think that I had the case when the route.to was "null" so there was an error. But I can try to remove it and check again.